docs/loadflow/parameters.md

```diff
@@ -140,6 +140,12 @@ If set to false, any existing voltage remote control is converted to a local con
 according to the nominal voltage ratio between the remote regulated bus and the equipment terminal bus.  
 The default value is `true`.
 
+**maxVoltageRemoteControlDistance**
+Defines the maximum distance allowed between the remote regulated bus and the voltage controller (requires `voltageRemoteControl` to be set to `true`).
+This distance is measured in number of branches separating the two buses. If the distance is higher than this parameter, then the voltage remote control is converted to a local control, rescaling the target voltage according to the nominal voltage ratio between the remote regulated bus and the equipment terminal bus.
+If `maxVoltageRemoteControlDistance` is set to `0`, there is no distance checking.
+The default value is `0`.
+

```

Comment:
I'm not sure of the default value.
Should we privilege backward compatibility (default = 0)  or robustness (default = 2 ) ?

Reply:
Yes default to 2 would be a good start.

Reply:
I had to modify one unit test (`NotSameNumberVariableEquationIssueTest.test`) with this default value because there are buses that have a distance of 4. I prefered to directly modify the values of the assert instead of changing the parameter in the test which would have made it less clear.